File: F/2221-find-triangular-sum-of-an-array/main.go
package main

func triangularSum(nums []int) int {
	for i := 0; i < len(nums); i++ {
		for j := 0; j < len(nums)-1-i; j++ {
			nums[j] = (nums[j] + nums[j+1])%10
		}
	}
	return nums[0] %10
}

File: F/2221-find-triangular-sum-of-an-array/main.rs
impl Solution {
  pub fn triangular_sum(mut nums: Vec<i32>) -> i32 {
    for i in 0..nums.len() {
      for j in 0..(nums.len() - 1 - i) {
        nums[j] += nums[j + 1];
        nums[j] %= 10;
      }
    }
    nums[0] % 10
  }
}
